presto-main/src/test/java/com/facebook/presto/sql/planner/plan/TestAssingments.java

```diff
@@ -14,22 +14,22 @@
 package com.facebook.presto.sql.planner.plan;
 
 import com.facebook.presto.spi.relation.VariableReferenceExpression;
-import com.google.common.collect.ImmutableCollection;
 import org.testng.annotations.Test;
 
 import static com.facebook.presto.spi.type.BigintType.BIGINT;
 import static com.facebook.presto.sql.planner.iterative.rule.test.PlanBuilder.assignment;
+import static com.facebook.presto.sql.relational.Expressions.variable;
 import static com.facebook.presto.sql.tree.BooleanLiteral.TRUE_LITERAL;
 import static org.testng.Assert.assertTrue;
 
 public class TestAssingments
 {
     private final Assignments assignments = assignment(new VariableReferenceExpression("test", BIGINT), TRUE_LITERAL);
 
-    @Test
+    @Test(expectedExceptions = RuntimeException.class)
```

Comment:
What happens to the test?

Reply:
After this is moved to SPI there's no `ImmutableCollection` anymore, hence this needs to be fixed. This should be part of the previous commit. Otherwise the previous commit won't pass tests.

Reply:
Yes, it is before moving anything into SPI :) 

The test here assumes we will return a Guava `ImmutableCollection` instead of checking the behavior of the return `container`.

Reply:
Let me reorder the commit. It seems like messed up by github.